FlagsmithClient/src/main/java/com/flagsmith/entities/IdentityAndTraits.kt

```diff
@@ -4,5 +4,6 @@ import com.google.gson.annotations.SerializedName
 
 data class IdentityAndTraits(
     @SerializedName(value = "identifier") val identifier: String,
-    @SerializedName(value = "traits") val traits: List<Trait>
+    @SerializedName(value = "traits") val traits: List<Trait>,
+    val transient: Boolean = false
```

Comment:
Why do the other attributes here have the `@SerializedName` decorator but not the newly added `transient` attribute?